Travis build is broken

The Travis build is broken because of changes to ol3. 

The error is:

```
node buildtools/build.js buildtools/ngeo.json dist/ngeo.js
info ol Parsing dependencies
ERR! Unsatisfied dependency "ol.ext.rbush" in script: /home/travis/build/elemoine/ngeo/node_modules/openlayers/src/ol/structs/rbush.js
make: *** [dist/ngeo.js] Error 1
```

--- a/buildtools/example.json
+++ b/buildtools/example.json
@@ -2,6 +2,7 @@
   "exports": [],
   "src": [
     "node_modules/openlayers/src/**/*.js",
+    "node_modules/openlayers/build/ol.ext/*.js",
     "src/**/*.js"
   ],
   "compile": {

--- a/buildtools/examples-all.json
+++ b/buildtools/examples-all.json
@@ -2,6 +2,7 @@
   "exports": [],
   "src": [
     "node_modules/openlayers/src/**/*.js",
+    "node_modules/openlayers/build/ol.ext/*.js",
     "src/**/*.js"
   ],
   "compile": {

--- a/buildtools/ngeo-simple.json
+++ b/buildtools/ngeo-simple.json
@@ -2,6 +2,7 @@
   "exports": ["*"],
   "src": [
     "node_modules/openlayers/src/**/*.js",
+    "node_modules/openlayers/build/ol.ext/*.js",
     "src/**/*.js"
   ],
   "compile": {

--- a/buildtools/ngeo-whitespace.json
+++ b/buildtools/ngeo-whitespace.json
@@ -2,6 +2,7 @@
   "exports": ["*"],
   "src": [
     "node_modules/openlayers/src/**/*.js",
+    "node_modules/openlayers/build/ol.ext/*.js",
     "src/**/*.js"
   ],
   "compile": {

--- a/buildtools/ngeo.json
+++ b/buildtools/ngeo.json
@@ -2,6 +2,7 @@
   "exports": ["*"],
   "src": [
     "node_modules/openlayers/src/**/*.js",
+    "node_modules/openlayers/build/ol.ext/*.js",
     "src/**/*.js"
   ],
   "compile": {

